test/cases/configuration/README.md

```diff
@@ -1,16 +1,14 @@
 These tests verify that the user can configure the Datadog module using the
-`datadog` configuration directive, and that if they don't, a suitable default
-configuration is chosen.
+`datadog_*` configuration directives, and that if they don't, a suitable default
+options are chosen.
 
-There are three classes of configuration scenarios:
+The following behaviors are tested:
 
-1. The user explicitly configured tracing using the `datadog` directive.
-2. The user implicitly configured tracing by using a directive that is
-   overridden by the Datadog module, such as `proxy_pass`.
-3. The user implicitly configured tracing _without_ using any overridden
-   directives.  This is probably rare, but it is what would happen if nginx's
-   default configuration had the Datadog module added to it (i.e. a static
-   content server). 
+- Certain `datadog_*` directives are allowed as direct children in the `http`
```

Comment:
Fun fact, github markdown's numbered lists don't care that much about the numeric value, as long as the first one is `1.`
The rest can be `1.`, `42.` or whatever and it'll renumber it. Just in case you wanted to keep it a numbered list without fussing over the sequence.

Reply:
Thanks. That's true of all markdown, and I always found it odd. It's a nice feature if you care about the rendered output.